hub/management/commands/import_areas.py

```diff
@@ -0,0 +1,39 @@
+import json
+from django.core.management.base import BaseCommand
+
+from time import sleep
+from utils import mapit
+from hub.models import Area
+
+
+class Command(BaseCommand):
+    help = "Import basic area information from MaPit"
```

Comment:
10 bonus points for the archaic capitalisation of MapIt. :+1:

I wonder whether `README.md` should explain that you’ll need to run `./manage.py import_areas` after you’ve started the docker container?

Reply:
Might also want to let them know it’ll take at least 10 minutes to run ;-)